README.md

```diff
@@ -1,32 +1,117 @@
-# dp-files-api
+# DP Files API
 
-An API for handling file meta data
+## Introduction 
+The Files API is part of the [Static Files System](https://github.com/ONSdigital/dp-static-files-compose).
```

Comment:
Link to a private repo from a public one - I suppose that it can't be helped, but...

Reply:
Is there any reason is could not be public?